test/test_unit.jl

```diff
@@ -1633,6 +1635,37 @@ end
     @test mesh.boundary_faces[:entire_boundary] == [1, 2]
 end
 
+@testset "PERK Single p2 Constructors" begin
+    path_coeff_file = mktempdir()
+    Trixi.download("https://gist.githubusercontent.com/DanielDoehring/8db0808b6f80e59420c8632c0d8e2901/raw/39aacf3c737cd642636dd78592dbdfe4cb9499af/MonCoeffsS6p2.txt",
+                   joinpath(path_coeff_file, "gamma_6.txt"))
+
+    ode_algorithm = Trixi.PairedExplicitRK2(6, path_coeff_file)
```

Comment:
This is probably causing the error

https://github.com/trixi-framework/Trixi.jl/actions/runs/8950168743/job/24585251161?pr=1908#step:7:1372